application/leds.c

```diff
@@ -0,0 +1,68 @@
+/* Copyright Ville Hiltunen 2018 <hiltunenvillej@gmail.com>
+ *
+ * All code here is public domain.
+ */
+#include <stdio.h>
+#include <string.h>
+
+#include "thread.h"
+#include "leds.h"
+
+static kernel_pid_t led_pid;
+
+void leds_set_color(color_rgba_t led_color){
+    /*
+     * The apa102 driver wants a color_rgba_t input. Problem is,
+     * even though we have it, RIOT messages are uint32_t, so we have
+     * cast the data type before sending.
+     */
+    msg_t m;
+    //Message needs to be set to 0. Otherwise garbage appears.
```

Comment:
Please remove all comments like this.  This does not add anything to the knowledgeable programmer.